src/components/UI/button/Button.test.tsx

```diff
@@ -0,0 +1,50 @@
+import Button from './Button';
+import { render, screen } from '@testing-library/react';
+import styles from './Button.module.css';
+import userEvent from '@testing-library/user-event';
+
+describe('Button', () => {
+  const BUTTON_NAME = 'Button';
```

Comment:
```suggestion
const ROLE_OPTIONS = { name: 'Button'};

describe('Button', () => {
```
You can use it in every test case